### DISCUSSION TITLE
Getting error when reloading form

### DISCUSSION THREAD

**Discussion by pankajdurgapur** [weareseeed/react-square-web-payments-sdk#39] [Help] [answered]
I have integrated this library in the next.js project and it is working fine when next.js routing navigating page and loading payment form but when manually reloading page getting error and second error is that sometimes payment form loading trice. Please see in attached screen shot. I would like to appreciate if some one will help me here. I don't think so this **react-square-web-payments-sdk": "2.2.3"** is compatible with next.js using typescript.
First error : in the image 876 and the second error : image id 821
Reference image: 
<img width="298" alt="image" src="https://user-images.githubusercontent.com/101563400/161251894-c2c87756-8e8a-4bfe-9113-1b3c7f82fb66.png">
<img width="301" alt="image" src="https://user-images.githubusercontent.com/101563400/161252094-ca4646ba-a11e-461f-afbc-a451d3585885.png">
<img width="386" alt="image" src="https://user-images.githubusercontent.com/101563400/161554297-0bdb01b6-4677-4f98-b758-6063362c483f.png">

<img width="876" alt="image" src="https://user-images.githubusercontent.com/101563400/161252324-8822afe8-1102-4745-ad61-99b12a15209d.png">

**Comment by danestves** [comment]
Hi @pankajdurgapur are you using the `2.5.0` version?

**Comment by pankajdurgapur** [comment]
@danestves Appreciate for your response .  I have updated package as you can see be below but still am getting same issue in the browser and project build time.

<img width="384" alt="image" src="https://user-images.githubusercontent.com/101563400/161489421-42380075-a4a1-4d85-add6-85f1274fa8d9.png">
<img width="728" alt="image" src="https://user-images.githubusercontent.com/101563400/161489458-032b3dbe-7376-461a-8cbc-b9f7b1fc1ddd.png">
<img width="595" alt="image" src="https://user-images.githubusercontent.com/101563400/161523870-80c67041-f154-4aa1-b572-883436fde4f7.png">

**Reply by danestves** [reply]
Hi @pankajdurgapur thank you for your patience, we have addressed this issue, and we are working on https://github.com/weareseeed/react-square-web-payments-sdk/pull/47 to fix it, hopefully this week the v3 will have landed

**Comment by danestves** [comment, marked as answer]
Hi @pankajdurgapur please try with the v3 of the wrapper that is out now https://github.com/weareseeed/react-square-web-payments-sdk/releases/tag/v3.0.0
